common/djangoapps/third_party_auth/strategy.py

```diff
@@ -29,9 +29,10 @@ def setting(self, name, default=None, backend=None):
             setting 'name' is configured via LTIProviderConfig.
         """
         if isinstance(backend, OAuthAuth):
-            provider_config = OAuth2ProviderConfig.current(backend.name)
-            if not provider_config.enabled_for_current_site:
+            providers = [p for p in Registry.displayed_for_login() if p.backend_name == backend.name]
+            if not providers:
                 raise Exception("Can't fetch setting of a disabled backend/provider.")
+            provider_config = providers[0]
```

Comment:
I think upstream has fixed this issue in a different way, so this problem should disappear in the next release. That is a larger fix and harder to backport. So the note here is that this will not be upstreamable.

Reply:
@xitij2000, any pointers/references to the PR/fix? If this can be removed in the next release of Open edX (say Juniper or Koa), is there a problem in retaining this code drift till we get the fix from upstream?

Reply:
I don't think so. Here is that PR: https://github.com/edx/edx-platform/pull/20070
I think backporting that will be too much effort at this point of time.